ecs-cli/modules/utils/compose/reconcile_container_def.go

```diff
@@ -123,6 +123,15 @@ func reconcileContainerDef(inputCfg *adapter.ContainerConfig, ecsConDef *Contain
 		if err != nil {
 			return nil, err
 		}
+
+		if ecsConDef.Gpu != "" {
```

Comment:
Will `ecsConDef.Gpu` ever be nil?

Reply:
From my understanding, `ecsConDef.Gpu` matches the `gpu` field in `ecs-params.yml` if the field exists.  If the field is not in the container definition then it defaults to its [zero value](https://tour.golang.org/basics/12) which is the empty string "". (I tested this)  

So I don't think there can be a scenario where it's nil. I tested the scenario where there